tuf/api/metadata.py

```diff
@@ -415,6 +441,50 @@ def remove_key(self, role: str, keyid: str) -> None:
             del self.keys[keyid]
 
 
+class MetadataInfo:
+    """A container with information about a particular metadata file.
+    Instances of MetadataInfo are used as values in a dictionary called
+    "meta" in Timestamp and Snapshot.
+
+    Attributes:
+        version: An integer indicating the version of the metadata file.
+        length: An optional integer indicating the length of the metadata file.
+        hashes: A optional dictionary containing hash algorithms and the
+            hashes resulting from applying them over the metadata file.::
+
+                'hashes': {
+                    '<HASH ALGO 1>': '<METADATA FILE HASH 1>',
+                    '<HASH ALGO 2>': '<METADATA FILE HASH 2>',
+                    ...
+                }
+
+    """
+
+    def __init__(self, version: int, length: Optional[int] = None,
+            hashes: Optional[JsonDict] = None) -> None:
+        self.version = version
+        self.length = length
+        self.hashes = hashes
+
+
+    def __eq__(self, other: 'MetadataInfo') -> bool:
+        """Compare objects by their values instead of by their addresses."""
```

Comment:
Nit: by default Python compares `object`s by their identity, which is [_like_ the address](https://docs.python.org/3/reference/datamodel.html?highlight=identity) but may not be. Can we use the Pythonic term identity here?

Reply:
Yep, didn't know about that.